projects/demo/src/modules/components/editor/editor.template.html

```diff
@@ -198,6 +189,17 @@ <h4>Text:</h4>
                 Allowed edit tools
             </ng-template>
         </tui-doc-documentation>
+        <tui-doc-documentation heading="CSS customization">
+            <ng-template
+                i18n
+                documentationPropertyName="style.max-height.px"
```

Comment:
nit: let's use camelCase for those, looks more natural to me and I believe we typically write it in camelCase, because it's kind of like the property on the style object. This comment applies to all such cases.

Reply:
fixed